Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/bebit/python-mecab-builder-dev:pr-14 as builder
+FROM ghcr.io/bebit/python-mecab-builder-dev:pr-15 as builder
```

Comment:
I think this change is unnecessary. (If there is some other changes to the branch in the future, this change will cause problem and this is very minor and will be replaced when releasing).